astropy/utils/console.py

```diff
@@ -365,7 +365,7 @@ def strip_ansi_codes(s):
     """
     Remove ANSI color codes from the string.
     """
-    return re.sub('\033\[([0-9]+)(;[0-9]+)*m', '', s)
+    return re.sub(r'\033\[([0-9]+)(;[0-9]+)*m', '', s)
```

Comment:
ok, will change this to a interpreted string with escaped `\\` again. Thanks for testing 👍